README.rst

```diff
@@ -30,44 +28,44 @@ Plugins List
     .. image:: https://raw.githubusercontent.com/eduNEXT/frontend-plugins-communications/main/screenshots/individual_learners_plugin.png
         :alt: communications-app-individual-emails Image
 
-**communications-app-instructions-pro-freading**
-   :NPM Package: `communications-app-instructions-pro-freading <https://www.npmjs.com/package/@edunext/plugins-communications-app-instructions-pro-freading>`_
+**communications-app-instructions-proof-reading**
+   :NPM Package: `communications-app-instructions-proof-reading <https://www.npmjs.com/package/@edunext/plugins-communications-app-instructions-proof-reading>`_
```

Comment:
you shouldn't change the package, that one doesn't exits
https://www.npmjs.com/package/@edunext/plugins-communications-app-instructions-proof-reading

Reply:
Yes, but pro-freading isn't a typo? what does pro-freading mean?